Updated CM README.md (#49757)
Azure/azure-sdk-for-net: sdk/cloudmachine/README.md

# Azure Projects client library for .NET

Azure.Projects is an set of libraries and tools for rapid application development on Azure.
If you cannot get an app running in 10 minutes, let us know that we failed!

> These libraries and tools make it easy to get apps off the ground, by:

Reviewer: ```suggestion
These libraries and tools make it easier to start building an application, by:
```

MS Docs guidelines suggest avoiding words like "easy" because it is subjective and may cause people who don't feel this is easy to feel negatively.

Author: I have to say I am not thrilled about this suggestion. It reads like hedging bets.

Reviewer: What's easy for Person A may be difficult for Person B.    If Person B struggles with this, the tone of the guide implies "this should be easy.  You're not good at this and should feel bad."

I'll look for the official guideline doc that discusses when I get a moment, but @scottaddie may know where it is off the top of his head.

Reviewer: Conversely, you could frame as aspirational and remove the connotation.  Something like "The goal of these libraries is to make it easy to...."

> - relying on opinionated defaults whenever possible.
> - using the Azure.Provisioning for provisioning resources in code (in C#).
> - exposing simplified APIs for the most commonly used Azure services.

Reviewer: ```suggestion
- Relying on opinionated defaults whenever possible.
- Using the Azure.Provisioning libraries for provisioning resources in code (in C#).
- Exposing simplified APIs for the most commonly used Azure services.
```

> At the same time, Azure.Projects supports break-glass scenarios where, if needed, you can override the defaults, use powerful tools (like bicep, azd), or the full featured Azure  SDK libraries. 

Reviewer 2: ```suggestion
At the same time, Azure.Projects supports break-glass scenarios where, if needed, you can override the defaults, use powerful tools (like bicep, azd), or the full featured Azure SDK libraries. 
```

> In other words, Azure.Projects will never block you from using the full power of the Azure platform, if you choose to. 

Reviewer 2: ```suggestion
In other words, Azure.Projects provides smart, simplified APIs but will never block you from using the full power of the Azure platform, if you choose to. 
```

## Getting started

### Prerequisites

* You must have an [Azure subscription](https://azure.microsoft.com/free/dotnet/).
* You must have [.NET 8](https://dotnet.microsoft.com/download) (or higher) installed
* You must have [Azure Developer CLI](https://learn.microsoft.com/azure/developer/azure-developer-cli/install-azd) (azd) installed
* You must be logged into Azure Developer CLI (`azd auth login`)

### Walkthrough

> #### Create C# Project

Reviewer 2: ```suggestion
#### Create a console project
```

In a command line window type

```dotnetcli
mkdir cmdemo
cd cmdemo
dotnet new console
```

#### Install Required Packages

```dotnetcli
dotnet add package Azure.Projects --prerelease
dotnet add package Azure.Projects.Provisioning --prerelease
dotnet add package Azure.Projects.AI --prerelease
```

#### Write, Provision, and Run the Application

Open `Program.cs` file and change the code to the following

> ```csharp

Reviewer: Please make all code samples snippets.  We generally enforce this for all libraries so that all sample code is at least compile-checked nightly.   No objection if you'd like to defer that work to get this out, but if so please file an issue to track.

Author: will do in a subsequent PR

using Azure.AI.OpenAI;
using Azure.Projects;
using Azure.Projects.Tooling;
using OpenAI.Chat;

ProjectInfrastructure infrastructure = new();

> infrastructure.AddFeature(new OpenAIChatFeature("gpt-35-turbo", "0125"));

Reviewer 2: We should new a newer model - I believe this one is deprecated.
